events_task/test/my_app_web/controllers/confirmation_controller_test.exs

```diff
@@ -54,20 +52,16 @@ defmodule MyApp.ConfirmationControllerTest do
         build_conn()
         |> post(confirmation_path(conn, :create, %{"event_id" => event.id, "user_id" => user.id}))
 
-      assert conn.private[:plug_session]["phoenix_flash"] == %{
-               "info" => "Confirmed successfully."
-             }
+      assert get_flash(conn, :error) == nil
+      clear_flash(conn)
```

Comment:
It doesn't work like that - in elixir you can't mutate a variable inside a function.
`conn = clear_flash(conn)`
But after other changes it looks like you don't need to clear it anymore.